nx_neptune/instance_management.py

```diff
@@ -499,21 +501,31 @@ def _start_export_task(
         s3_destination (str): The S3 destination location (e.g., 's3://bucket-name/prefix/')
         role_arn (str): The IAM role ARN with permissions to write to the S3 bucket
         kms_key_identifier (str): KMS key ARN for encrypting the exported data
-        filetype (str): CSV
+        filetype (str, optional): The format of the export data. Defaults to "CSV".
+        export_filter (dict, optional): Filter criteria for the export. Defaults to None.
```

Comment:
link to documentation? I think we should include some of the documentation included in https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/neptune-graph/client/start_export_task.html - at least the bare minimum to get the user started, and they can look up the details if needed.